src/LibraryManager.Vsix/UI/Controls/TextBoxWithSearch.xaml.cs

```diff
@@ -231,7 +231,7 @@ private void SearchTextBox_TextChanged(object sender, TextChangedEventArgs e)
                 {
                     CompletionSet completionSet = await ViewModel.GetCompletionSetAsync(SearchTextBox.CaretIndex);
 
-                    if (completionSet.Equals(null) || !completionSet.Completions.Any())
+                    if (completionSet.Completions == null || !completionSet.Completions.Any())
```

Comment:
Should we still have a null check on `completionSet`?  Maybe do `if (completionSet?.Completions is null || ...)`?

Reply:
never mind, saw that it's a struct, so null check isn't needed/valid.